src/random_test.cpp

```diff
@@ -340,13 +344,25 @@ std::string Column::rand_value() {
     return std::to_string(rand_int(rec));
     break;
   case (COLUMN_TYPES::FLOAT):
-    static auto rec1 = opt_int(INITIAL_RECORDS_IN_TABLE);
-    return std::to_string(rand_float(rec1));
+  {
+    int rec1 = opt_int(INITIAL_RECORDS_IN_TABLE);
+    rec1 /= 100;
+    std::ostringstream out;
+    out << std::fixed;
+    out << std::setprecision(2) << rand_float(rec1);
+    return out.str();
     break;
+  }
   case (COLUMN_TYPES::DOUBLE):
-    static auto rec2 = opt_int(INITIAL_RECORDS_IN_TABLE);
-    return std::to_string(rand_float(rec2));
+  {
+    int rec2 = opt_int(INITIAL_RECORDS_IN_TABLE);
+    rec2 /= 100;
+    std::ostringstream out;
+    out << std::fixed;
+    out << std::setprecision(5) << rand_float(rec2);
```

Comment:
with set precision(5). there will be very hard to find records of double in update/delete queries. you need a better division.

Reply:
Fixed